keel-core/src/main/kotlin/com/netflix/spinnaker/keel/veto/unhappy/UnhappyVeto.kt

```diff
@@ -44,7 +44,8 @@ class UnhappyVeto(
   private val diffFingerprintRepository: DiffFingerprintRepository,
   private val unhappyVetoRepository: UnhappyVetoRepository,
   private val dynamicConfigService: DynamicConfigService,
-  @Value("veto.unhappy.waiting-time")
+
+  @Value("\${veto.unhappy.waiting-time:PT10M}") // tests fail if default not specified here
```

Comment:
The duplication of `PT10M` is gross, but if it's not there, multiple tests will fail with the following exception:

```
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'vetoController' defined in file [/Users/lhochstein/netflix/spinnaker/workspace/keel/keel-web/out/production/classes/com/netflix/spinnaker/keel/rest/VetoController.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'unhappyVeto' defined in file [/Users/lhochstein/netflix/spinnaker/workspace/keel/keel-core/out/production/classes/com/netflix/spinnaker/keel/veto/unhappy/UnhappyVeto.class]: Unexpected exception during bean creation; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'veto.unhappy.waiting-time' in value "${veto.unhappy.waiting-time}"
```